packages/create-webstone-app/template/plugin/structure/packages/cli/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+	"useTabs": true,
+	"singleQuote": true,
```

Comment:
Same as above, is this intended?

Reply:
If I'm not wrong I copied the one that gets delivered when crating a new sveltekit app that has prettier configured 🤔

Reply:
:+1: If that's the case, let's keep it so `cli` and `web` plugins have consistent formatting.